analytics/population/nc_updated_pop_comparison.ipynb

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pop_diff_a = pop_diff[pop_diff['pop_diff(%)'] &lt;= 10]
</pre><p>Let's give these variables more meaningful names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Renamed the variables. </p>